test/xUnit/csharp/test_Subsystem.cs

```diff
@@ -86,8 +126,7 @@ public static void RegisterSubsystem()
 
                 // Now validate the all-subsystem-info collection.
                 ReadOnlyCollection<SubsystemInfo> ssInfos = SubsystemManager.GetAllSubsystemInfo();
-                Assert.Single(ssInfos);
-                Assert.Same(ssInfos[0], ssInfo);
+                VerifyAllSubsystemInfo(ssInfos, ssInfo, crossPlatformDscInfo);
```

Comment:
Let's remove the checks on all-subsystem-info here. It's already tested in the above `GetSubsystemInfo` test.

Reply:
updated.